webapp/src/views/IntegrationsAdminView.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <BasePage :title="$t('integrationsadmin.IntegrationSettings')" :isLoading="dataLoading">
+        <BootstrapAlert v-model="showAlert" dismissible :variant="alertType">
+            {{ alertMessage }}
+        </BootstrapAlert>
+
+        <form @submit="saveIntegrationsConfig">
+            <CardElement :text="$t('integrationsadmin.goecontroller')" textVariant="text-bg-primary">
+                <InputElement
+                    :label="$t('integrationsadmin.goecontrollerEnabled')"
+                    v-model="integrationsConfig.goe_ctrl_enabled"
+                    type="checkbox"
+                    wide
+                    :tooltip="$t('integrationsadmin.goecontrollerEnabledHint')"
+                />
+
+                <InputElement
+                    v-show="integrationsConfig.goe_ctrl_enabled"
+                    :label="$t('integrationsadmin.goecontrollerEnableHomeCategory')"
+                    v-model="integrationsConfig.goe_ctrl_publish_home_category"
+                    type="checkbox"
+                    wide
+                    :tooltip="$t('integrationsadmin.goecontrollerEnableHomeCategoryHint')"
+                />
+
+                <InputElement
+                    v-show="integrationsConfig.goe_ctrl_enabled"
+                    :label="$t('integrationsadmin.goecontrollerHostname')"
+                    v-model="integrationsConfig.goe_ctrl_hostname"
+                    type="text"
+                    placeholder="go-econtroller_XXXXXX"
```

Comment:
This is the hostname format, the X's being the serial number, usually starting with 9 as the first digit.